docs/pages/blog/first-look-at-joy.md

```diff
@@ -0,0 +1,133 @@
+These small details mean the components adapt to different scenarios, which saves considerable time by avoiding manual adjustments.
+
+<img src="/static/blog/first-look-at-joy/component-integration.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Screenshot of two text inputs, one being native from Joy UI and another with border-radius customized" />
+
+<p class="blog-description">When customizing the input's border radius, the icon button inside of it adapts automatically.</p>
+
+```jsx
+<Input
+  placeholder="password"
+  endDecorator={<IconButton size="sm"><Visibility /></IconButton>}
+/>
+
+<Input
+  size="lg"
+  placeholder="password"
+  endDecorator={<IconButton><Visibility /></IconButton>}
+  sx={{ '--Input-radius': '24px' }}
+/>
+```
+
+Joy UI doesn't sacrifice the customization experience in order to have these automatic adjustments.
+You're still able to override the style completely via the usual CSS overrides, or even adjust the predefined CSS variables with the `sx` prop.
+
+### Perfect dark mode, for server-side rendering
+
+Joy UI provides a simple and effective way to prevent UI flicker when users refresh or re-enter the page when dark mode is enabled.
+What's more, it takes care of the synchronization between browser tabs with a single line of code, which allows developers to configure attributes and settings as required.
+
+### Unlimited color schemes
+
+Joy UI not only allows for simple toggling between dark and light mode but also provide the right tooling so you can provide your users with multiple color schemes.
+Within each mode, you'll be able to have as many as you want－powered by CSS variables.
+
+## When will I be able to use it?
+
+A significant part of the core infrastructure for Joy UI has already been developed, and we've been working for the last couple of months on adding more and more components.
+The package is already [available at npm](https://www.npmjs.com/package/@mui/joy) but until we get to a reasonable number of components, and with sufficient documentation, it remains as _initial_ work in progress.
+
+It should, though, be ready for more mature experimentation and testing most likely dureing the second half of 2022.
+Meanwhile, we've developed a couple of templates that showcase what Joy looks like out of the box, and to allow you to explore it further.
+Follow the templates' CodeSandbox or live demo links to experiment with Joy UI for yourself.
+
+:::info
+👉 They are also available on the docs [Templates page](/joy-ui/getting-started/templates/).
+:::
+
+### Email
+
+<img src="/static/blog/first-look-at-joy/email.png" style="width: 796px; margin-top: 8px; margin-bottom: 16px;" alt="Screenshot of an email mock up application built with Joy UI" />
+
+- [Live demo](/joy-ui/getting-started/templates/email/)
+- [CodeSandbox](https://codesandbox.io/s/8gtttr?file=/App.tsx)
```

Comment:
Some of the images are broken on codesandbox, maybe we should use some public images, or duplicate them in the sandbox.

Reply:
cc @siriwatknp